estate/views/estate_inherited_user_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```

Comment:
data is only useful to split between updatable and noupdate data. Here, everything is updatable, so no need to define it